eventdiscovery-sdk/src/main/res/layout/schedjoules_view_divider.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I remembered as if I had some problem with simply using a View as root, but now I tried and it works, so I changed it.